docs/loadjdbc.adoc

```diff
@@ -224,3 +228,266 @@ CREATE (t)-[:GENRE]->(g);
 ----
 Added 63213 labels, created 63213 nodes, set 182413 properties, created 119200 relationships, statement executed in 40076 ms.
 ----
+
+== Postgres Example
+
+== Describe EMPLOYEE_DETAILS;
+
+[source,sql]
+----
+Describe Employee_Details
+
+\d+ EMPLOYEE_DETAILS
+
+
++----------+---------------+------------+-----------+-------------+----------------+
+| Column   | Type          | Modifiers  | Storage   | Stats target| Description	   |
++----------+---------------+------------+-----------+-------------+----------------+
+| id       | integer       | not null   | plain     |             |                |
+| name     | text          | not null   | extended  |             |                |
+| age      | integer       | not null 	| plain     |             |
+| address  | character(50) | 	  	| extended  |             |                |
+| salary   | real          |   		| plain     |             |                |
++----------+---------------+------------+-----------+-------------+----------------+
+
+  Indexes:
+	"employee_details_pkey" PRIMARY KEY, btree(id)
+
+
+
+select count(*) from EMPLOYEE_DETAILS;
++----------+
+| count(*) |
++----------+
+|       50 |
++----------+
+
+(1 row)
+
+----
+
+Download the https://jdbc.postgresql.org/download.html[PostgresSQL JDBC Driver], and put it into your `$NEO4J_HOME/plugins` directory.
+
+Now Retart the server.
+
+== Load JDBC Examples
+
+.Load the JDBC driver
+[source,cypher]
+----
+cypher CALL apoc.load.driver('org.postgresql.Driver');
+----
+
+image:https://raw.githubusercontent.com/neo4j-contrib/neo4j-apoc-procedures/master/docs/img/apoc.load.jdbc.postgres.jpg[width=800]
+
+
+.Count rows in products table
+[source,cypher]
+----
+with "jdbc:postgresql://localhost:5432/testdb?user=postgres&password=postgres" as url
+CALL apoc.load.jdbc(url,"employee_details") YIELD row
+RETURN count(*);
+----
+
+----
++----------+
+| count(*) |
++----------+
+|       50 |
++----------+
+----
+
+.Return row from products table
+[source,cypher]
+----
+with "jdbc:postgresql://localhost:5432/testdb?user=postgres&password=postgres" as url
+CALL apoc.load.jdbc(url,"employee_details") YIELD row
+RETURN row LIMIT 4;
+----
+
+----
+╒══════════════════════════════╕
```

Comment:
use a wide variant of the table, there is a little slider on the bottom right to pull the table wide